Chronometer in Android.md

```diff
@@ -0,0 +1,133 @@
+_Android ChronoMeter is user interface control which shows timer in the view. We can easily start up or down counter with base time using the chronometer widget. By default, start() method can assume base time and starts the counter.
+Generally, we can create use ChronoMeter widget in XML layout but we can do it programmatically also._
```

Comment:
this doesnt fixed the mentioned issue, hence this file is not required in this PR